packages/core/src/types/IImageVolume.ts

```diff
@@ -81,10 +81,13 @@ interface IImageVolume {
   destroy(): void;
 
   /** decache */
-  decache?: () => void;
+  decache?: (completelyRemove?: boolean) => void;
 
   /** */
   get imageCacheOffsetMap(): Map<string, any>;
+
+  /** updates volume internals  */
```

Comment:
The comment should be more generic - something like "mark the volume as having had the pixel data changed externally".  That explains what the call does, not how it does it or what it does internally to performt he operation.